fbw-a380x/src/systems/instruments/src/SD/Pages/Press/elements/LandingElevation.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import { useSimVar } from '@instruments/common/simVars';
 import { Position } from '@instruments/common/types';
+import { useArinc429Var } from '@instruments/common/arinc429';
 
 
 const LandingElevation: React.FC<Position> = ({ x, y }) => {
-    const [landingElev] = useSimVar('L:A32NX_FM1_LANDING_ELEVATION', 'feet', 100);
+    const landingElev = useArinc429Var('L:A32NX_FM1_LANDING_ELEVATION', 1000);
+    const autoMode = useSimVar('L:A32NX_OVHD_PRESS_MAN_ALTITUDE_PB_IS_AUTO', 'Bool', 1000);
 
-    const ldgElevValue = Math.round(landingElev / 50) * 50;
+    const ldgElevValue = Math.round(landingElev.value / 50) * 50;
 
     return (
         <>
-            <g id="LandingElevation">
+            <g id="LandingElevation" className={autoMode ? 'Show' : 'Hide'}>
                 <text className="F25 White LS1" x={x} y={y}>LDG ELEVN</text>
 
                 <text id="LandingElevation" className="F28 Green EndAlign" x={x + 260} y={y}>{ldgElevValue}</text>
```

Comment:
Should be amber XX when the SSM is invalid. Is that out of scope?

Reply:
Yes out of scope for this PR as this case is not handled correctly yet within the pressurization system either. I've added a TODO comment, should be an easy addition later on.